### PR TITLE
IDVA6-461 cancel authorisation - are you sure you want to cancel

### PR DESCRIPTION
**Pull request by mmodzelewski-ch**
Link to Jira:
https://companieshouse.atlassian.net/browse/IDVA6-461

Changes:
- Cancel authorisation page and its logic created
- Various changes to templates to replace hard coded values with provided by language files
- Minor fixes

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/views/layouts/shell.njk [hunk 1]
```diff
@@ -14,7 +14,7 @@
                 : 'js-enabled');
         </script>
         {% include "partials/cookie_consent_banner.njk" %}
-        <div role="navigation" aria-label="skip to main content">
+        <div role="navigation" aria-label="lang.skip_to_main_content">
             <a href="#main-page-content" class="govuk-skip-link">Skip to main content</a>
```
Comment (bmchugh-ch) [inline src/views/layouts/shell.njk:18]: Does this work okay without brackets like you have in default.njk?

**Review by bmchugh-ch — APPROVED** [review]

File: src/views/layouts/shell.njk (hunk 1 above)
Comment (mmodzelewski-ch) [reply src/views/layouts/shell.njk:18]: Well spotted. It's fixed now.

**Review by bmchugh-ch — APPROVED** [review]

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Pull request merged**